apps/server/src/utilities/network.service.ts

```diff
@@ -38,7 +38,13 @@ export class NetworkService implements OnApplicationShutdown {
         });
 
         this.mDNSServer.on('warning', (error) => {
-            this.logger.warn(`mDNS server warning: ${error.message}`);
+            if (/Cannot decode name \(.+\)/.test(error.message)) {
```

Comment:
Are you sure you want to use regex instead of .contains?

Reply:
Regex is more readable in my opinion, and this won't be run very often anyway.